test/functional/feature_proxy.py

```diff
@@ -143,6 +162,7 @@ def node_test(self, node, proxies, auth, test_onion=True):
             assert_equal(cmd.username, None)
             assert_equal(cmd.password, None)
         rv.append(cmd)
+        self.network_test(node, addr, network=NET_UNROUTABLE)
```

Comment:
bebd062eef3fa2d33c9216c368977dbedb70a443

Testing of `network=NET_UNROUTABLE` is actually skipped. Could we avoid such skipping?

Reply:
When I run `feature_proxy.py` with `-l debug`, or with the following patch, it doesn't seem to be skipped in my testing. LMK if I'm misunderstanding.
```diff
+++ b/test/functional/feature_proxy.py
@@ -101,6 +101,7 @@ class ProxyTest(BitcoinTestFramework):
         for peer in node.getpeerinfo():
             if peer["addr"] == addr:
                 assert_equal(peer["network"], network)
+                import pprint; pprint.pprint(network)
```